### ISSUE TITLE
blkid uuid's print order

### ISSUE THREAD

**Issue by dcegielka** [landley/toybox#34]
```
# ./toybox blkid | grep sda1 ; blkid | grep sda1                                          
/dev/sda1: LABEL="EFI" UUID="ed17-e367" TYPE="vfat"
/dev/sda1: LABEL="EFI" UUID="67E3-17ED" TYPE="vfat"
```

**Comment by technosaurus** [comment]
My original version attempted to put the "right" UUID formatting for each different filesystem... it looked really hacky because it was.  It was because it needed to be.  It needed to be because nobody ever came together and defined "UUID" with real specification, so every different filesystem used their own definition from 8 to 16 byte versions  to mixtures of 1, 2, 4 or 8 byte sets in big or little endian... its pretty ridiculous.   I never really asked why, but I assume Rob decided that a single unified definition of UUID with respect to printing made more sense because its simpler and still serves the same purpose for 99% of use cases.  I can see a possible issue where the initramfs has toybox blkid to mount stuff by UUID and the the rootfs has the bloated version, but have not seen it in any of my distro hopping experience.

**Comment by enh-google** [comment]
oh, i fixed this in passing with 9dc24d994efc3837e248ef8db9b3976e8775235e but didn't realize there was an existing bug for it.

rob: you can close this one.
